src/Cli/dotnet/Commands/Package/Add/Program.cs

```diff
@@ -67,25 +67,26 @@ public override int Execute()
 
     private void GetProjectDependencyGraph(string projectFilePath, string dgFilePath)
     {
-        var args = new List<string>();
-
-        // Pass the project file path
-        args.Add(projectFilePath);
+        var args = new List<string>
```

Comment:
Maybe something like:
```cs
List<string> args = 
[
      // Pass the project file path
      projectFilePath,
      
      // Pass the task as generate restore Dependency Graph file
      "-target:GenerateRestoreGraphFile",

      // Pass Dependency Graph file output path
      $"-property:RestoreGraphOutputPath=\"{dgFilePath}\"",

      // Turn off recursive restore
      $"-property:RestoreRecursive=false",

      // Turn off restore for Dotnet cli tool references so that we do not generate extra dg specs
      $"-property:RestoreDotnetCliToolReferences=false",

      // Output should not include MSBuild version header
      "-nologo"
];
```
?

(GH doesn't allow me to make suggestions on deleted lines apparently)

Reply:
Fixed.